docs/modules/panel_configuration/pages/index.adoc

```diff
@@ -19,22 +19,116 @@ This section describes the following types of panel configuration:
 
-Alarm Table:: Displays alarms in table format.
+* *Filter panel:* A panel that lets you specify data to include in other panels.
 +
-image::alarm-panel.png[Two tables: one displays unacknowledged alarms, and the other displays acknowledged alarms, 500]
+image::filter-panel.png["Blank fields in a Filter panel", 850]
 
-Filter Panel:: Panel that lets you specify data to include in other panels.
+* *Flow Histogram:* A bar chart that shows the distribution of flows data.
+The query for this panel must include an `asTableSummary()` transform, otherwise the panel will not display any data.
 +
-image::filter-panel.png[Blank fields in a filter panel, 500]
+image::flow-histogram.png["A Flow Histogram panel displaying a horizontal bar graph. The bar graph shows network flow data.", 850]
+
+For a panel to display data visualizations, you must define the following settings:
+
+* Datasource
+* Flow Type
+* Time Interval
+
+== Customize panels
+
+You can customize the data visualized on {page-component-title} panels by tweaking their settings and changing their datasources.
+The following procedures demonstrate how to set up each panel type to show data from the flows datasource.
+
+=== Alarm Histogram
+
+Follow these steps to customize an Alarm Histogram panel:
+
+. Enter editing mode and select an Alarm Histogram panel on your dashboard.
+. Choose a datasource from the *Data Source* list (in this example, *OpenNMS Entities*).
+. Choose a display interval from the *Time Interval* list at the top of the panel.
+. Expand the *Query Options* section and update the settings as desired.
+. Edit the query using the fields at the bottom of the panel:
+** Choose a data type to display from the *Select* list (in this example, *Alarms*).
+** Set the visualization conditions using the *Where* options.
+** Choose the order in which the data is visualized from the *Order By* list.
+** (Optional) Set a limit on the amount of data displayed.
+** (Optional) Turn on the *Featured Attributes* option.
+. Update the panel options under the Grouping section:
+** Select the *Grouping* type (in this example, *Severity*).
+This sets the method by which data is grouped in the panel.
+** Select the display orientation using the *Settings* list (in this example, *Vertical*).
+. Set other display options as desired.
+
+Using these settings, the Alarm Histogram panel displays the following visualization:
+
+image::alarm-histogram-example.png["Example of an Alarm Histogram panel in edit mode. Areas of note are outlined in red.", 850]
+
+=== Alarms Table
+
+Follow these steps to customize an Alarms Table panel:
 
-Flow Histogram:: Bar chart that shows the distribution of flow data.
-This panel expects the Flow Datasource query to have a `asTableSummary()` transform included.
+. Enter editing mode and select an Alarms Table panel on your dashboard.
+. Choose a datasource from the *Data Source* list (in this example, *OpenNMS Entities React*).
```

Comment:
```suggestion
. Choose a datasource from the *Data Source* list (in this example, *OpenNMS Entities*).
```
Ah the **React** text is during this migration we are working on, but it shouldn't be part of the documentation since it will be removed once we merged into develop.